frontend/public/locales/en/credentials.json

```diff
@@ -21,7 +21,7 @@
 
     "credentialsForm.title.add": "Add credentials",
     "credentialsForm.title.edit": "Edit credentials",
-    "credentialsForm.title.tooltip": "A credentials stores the access credentials and configuration information for creating clusters.",
+    "credentialsForm.title.tooltip": "A credential stores the access credentials and configuration information for creating clusters.",
```

Comment:
A credential stores the access credentials
credential => crendential[] ?

That's why I have been refering to them a credentials and not credential.